apps/checkout/src/sections/CheckoutForm/useCheckoutFinalize.ts

```diff
@@ -0,0 +1,53 @@
+import { pay as payRequest } from "@/fetch";
+import { useCheckout } from "@/hooks/useCheckout";
+import { useFetch } from "@/hooks/useFetch";
+import { extractMutationErrors } from "@/lib/utils";
+import { useErrors } from "@/providers/ErrorsProvider";
+import { useAuth, useAuthState } from "@saleor/sdk";
+import { FormData } from "./types";
+import omit from "lodash/omit";
```

Comment:
https://www.npmjs.com/package/lodash-es 
should automatically treeshake, no need for deep imports